### DISCUSSION TITLE
Setting Modifier Keys on Side Buttons

### DISCUSSION THREAD

**Discussion by dschibster** [linearmouse/linearmouse#472] [Q&A] [answered]
Hi there!

First of all, love the app. Finally my USB mouse feels the way I'm used to from my Windows Rig. 

I have one question: On Windows, I'm using X-Mouse Button Config to my back and forward mouse buttons to Modifier Keys in order to better hit some hotkeys in a certain game. I've seen that the Mouse Button Reconfig is currently limited to specific functions. Wondering if there is a way to circumvent that, or if that is on the horizon. 

Best wishes
Dennis

**Comment by lujjjh** [comment, marked as answer]
Hey Dennis,

Thank you for your feedback! I'm glad to hear that you're enjoying the app and that it's improving your mouse experience.

Regarding your question, I believe that the feature you're looking for is feasible to implement. I've created an feature request #473 so that you can subscribe and receive updates on the progress of this feature.

Best regards,
Jiahao Lu

**Reply by dschibster** [reply]
Thanks a lot! I have subscribed to the issue.

To be a little more detailed, either a single Modifier Button Press or a combination of a few in one press would be perfect. Right now, for example, I am using MB4 for CTRL and MB5 for ALT(so CMD)+CTRL, in order to hit chords of different modifiers with just one additional key on the keyboard.

Best of luck with the ongoing development. :)
